resources/asciidoctor/spec/creamped_include_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'added/extension'
```

Comment:
This filename is typo'd

Reply:
Yeah. We don't need it here. I'll drop it.

Reply:
No I mean literally the filename. You meant `cramped` but it is `creamped` :)

Reply:
Ah!

Reply:
Fixed!